__tests__/feeds.ts

```diff
@@ -2276,7 +2276,21 @@ describe('function feedToFilters', () => {
   it('should return filters for tags/sources based on the values from our data', async () => {
     loggedUser = '1';
     await saveFeedFixtures();
-    expect(await feedToFilters(con, '1', '1')).toMatchSnapshot();
+    const filters = await feedToFilters(con, '1', '1');
+    expect(filters.blockedContentCuration).toEqual([]);
+    expect(filters.excludeTypes).toEqual([]);
+    expect(filters.sourceIds).toEqual([]);
+    expect(filters.includeTags).toEqual(
+      expect.arrayContaining(['html', 'javascript']),
+    );
```

Comment:
In addition to `arrayContaining`  maybe also check length so we are sure no additional elements are added and test does not catch it?

Reply:
also below

Reply:
👍